src/main/java/core/basesyntax/dao/TransactionDaoImpl.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Transaction;
+import java.util.List;
+
+public class TransactionDaoImpl implements TransactionDao {
```

Comment:
it's a redundant class, you already have an implementation where you pass fileName and read content of the file